hw_diag/tests/test_blockchain_helium_height.py

```diff
@@ -18,6 +20,8 @@ class TestHelium(unittest.TestCase):
     the_response.json.return_value = get_data
     the_response.status_code = 200
 
+    os.environ['DEFAULT_TIMEOUT'] = '5'
```

Comment:
Can this be removed?

Reply:
Yeah that's true it's no longer needed, I have removed it